access/advanced/permit.md

```diff
@@ -63,10 +57,12 @@ Alter an existing **permission**.
 #### ❌ Raises
 * `SecurityRequestError`<br>
   Raises `SecurityRequestError` when the **Return Code** of a **Security Result** returned by IRRSMO00 is **NOT** equal to `0`.
+* `SegmentTraitError`<br>
+  Raises `SegmentTraitError` when the dictionary of **traits/attributes** supplied contains at least one unknown trait.
```

Comment:
```markdown
* `SegmentTraitError`<br>
  Raises `SegmentTraitError` when the dictionary of **Traits** provided contains at least one unknown trait.
```

Reply:
Sure, this is now done.

Reply:
Same change implemented for SegmentError

Reply:
I am also thinking that the following update should be made as well:

```markdown
* `SegmentTraitError`<br>
  Raises `SegmentTraitError` when the dictionary of **traits/attributes** provided contains one or more **unknown** traits.
```

Reply:
Fair enough, change made.